test/public/defaults.js

```diff
@@ -217,7 +217,7 @@ module.exports.waitForFirstRowToHaveId = async (page, id) => page.waitForSelecto
  * @return {Promise<*>} resolves once the navigation finished
  */
 const waitForNavigation = (page, navigateFunction) => Promise.all([
-    page.waitForNavigation({ timeout: 1500 }),
+    page.waitForNavigation({ timeout: 5000 }),
```

Comment:
You can remove the timeout at all because the default is set to 5000

Reply:
I think the default from Puppeteer is 30000ms, or do you mean something else?

Reply:
Done